Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -3750,6 +3733,7 @@ + (TSOutgoingMessage *)createFakeOutgoingMessage:(TSThread *)thread
                                     messageState:(TSOutgoingMessageState)messageState
                                      isDelivered:(BOOL)isDelivered
                                           isRead:(BOOL)isRead
+                                  isVoiceMessage:(BOOL)isVoiceMessage
```

Comment:
I think you want to pass this through right?

Reply:
Thanks!